exonum-java-binding/core/src/main/java/com/exonum/binding/core/storage/database/View.java

```diff
@@ -70,8 +72,9 @@ public boolean canModify() {
    *
    *  @throws IllegalStateException if the view is invalid (closed or nullptr)
    */
-  public long getViewNativeHandle() {
-    return super.getNativeHandle();
+  public NativeHandle getViewNativeHandle() {
```

Comment:
Technically this is a breaking change. As an alternative I can add a separate method for returning `NativeHandle`. Would it be better?